File: GDocs.Site/src/app/assinatura/models/documento-fi1548-ciencia-model.ts
import { CienciaUsuarioAprovacaoModel } from "src/app/saida-material-nota-fiscal/shared/models/ciencia-usuario-aprovacao-model";

export class DocumentoFI1548CienciaModel{
    cienciaId: number;
    documentoFI1548Id: number;
    documentoFI1548Numero: number;
    idTipoCiencia: number;
    tipoCiencia: string;
    idStatusCiencia: number;
    statusCiencia: string;
    idUsuario: string;
    nomeUsuario: string;
    observacao: string;
    flgAtivo: boolean;
    ciente: boolean;
    motivoSolicitacao: string;
    justificativa: string;
    fornecedor: string;
    moedaSimbolo: string;
    valorTotal: string;
    pagamento: string;
    motivoRejeicao: string;
    cienciaUsuarioAprovacao: CienciaUsuarioAprovacaoModel[];
}

File: GDocs.Api/src/Infra/CrossCutting/Models/DocumentoFI1548CienciaModel.cs
using ICE.GDocs.Infra.CrossCutting.Models.Enums;
using ICE.GDocs.Infra.CrossCutting.Models.SaidaMaterialNotaFiscal;
using System;
using System.Collections.Generic;
using System.Text;

namespace ICE.GDocs.Infra.CrossCutting.Models
{
    public class DocumentoFI1548CienciaModel
    {
        public int CienciaId { get; set; }
        public int DocumentoFI1548Id { get; set; }
        public int DocumentoFI1548Numero { get; set; }
        public int IdTipoCiencia { get; set; }
        public string TipoCiencia { get; set; }
        public int IdStatusCiencia { get; set; }
        public string StatusCiencia { get; set; }
        public Guid IdUsuario { get; set; }
        public string NomeUsuario { get; set; }
        public string Observacao { get; set; }
        public bool FlgAtivo { get; set; }
        public bool Ciente { get; set; } = false;
        public string MotivoSolicitacao { get; set; }
        public string Justificativa { get; set; }
        public string Fornecedor { get; set; }
        public string MoedaSimbolo { get; set; }
        public decimal ValorTotal { get; set; }
        public string Pagamento { get; set; }
        public string MotivoRejeicao { get; set; }
        public IEnumerable<CienciaUsuarioAprovacaoModel> CienciaUsuarioAprovacao { get; set; }
    }
}
